interface/backend/cases/models.py

```diff
@@ -10,6 +10,48 @@
 from rest_framework import serializers
 
 
+def django_enum(cls):
+    # decorator needed to enable enums in django templates
+    cls.do_not_call_in_templates = True
+    return cls
+
+
+class PleuralSpace(models.Model):
+    created = models.DateTimeField(default=timezone.now)
+
+    @unique  # ensures all variables are unique
+    @django_enum
+    class TypesOne(IntEnum):
+        NONE = 0
+        SMALL = 1
+        MODERATE = 2
+        LARGE = 3
+
+    @unique  # ensures all variables are unique
+    @django_enum
+    class TypesTwo(IntEnum):
+        NONE = 0
+        MILD = 1
+        MODERATE = 2
+        EXTENSIVE = 3
+
+    effusion = models.IntegerField(
+        choices=[(choice.value, choice.name.replace("_", " ")) for choice in TypesOne],
```

Comment:
Feels like we could abstact this :)

Reply:
Done, I've added the [`format_enum`](https://github.com/concept-to-clinic/concept-to-clinic/pull/185/commits/d4e5a767b0dc943a624d9ee577a5a1dbe990ac4d#diff-11ad5b67ec28e05935f1d6da87a683e8R10) function inside the [`enum.py`](https://github.com/concept-to-clinic/concept-to-clinic/pull/185/commits/d4e5a767b0dc943a624d9ee577a5a1dbe990ac4d#diff-11ad5b67ec28e05935f1d6da87a683e8) file.